content/TruenasApps/_index.md

```diff
@@ -235,6 +235,20 @@ If the GPU was previously isolated and/or assigned to a VM, a reboot could be re
 Restart the system then return to the **Resources Configuration** section of the application to see if expected devices are available.
 {{< /expand >}}
 
+## Installing NVIDIA Drivers
+Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must instead manually install drivers from the TrueNAS UI.
+
```

Comment:
When writing a procedure, start steps with the verb action. Remember parallel construction principles apply and procedures are about the action. You can add clarification after the action or just add the result of the action. See the suggested changes below.